source/client/process_impl.cc

```diff
@@ -455,6 +477,28 @@ bool ProcessImpl::runInternal(OutputCollector& collector, const std::vector<UriP
 
     Runtime::LoaderSingleton::get().initialize(*cluster_manager_);
 
+    std::list<std::unique_ptr<Envoy::Stats::Sink>> stats_sinks;
+    for (const envoy::config::metrics::v3::StatsSink& stats_sink : bootstrap.stats_sinks()) {
+      ENVOY_LOG(info, "loading stats sink configuration in Nighthawk");
+      auto& factory =
+          Envoy::Config::Utility::getAndCheckFactory<NighthawkStatsSinkFactory>(stats_sink);
+      stats_sinks.emplace_back(factory.createStatsSink(store_root_.symbolTable()));
+    }
+    for (std::unique_ptr<Envoy::Stats::Sink>& sink : stats_sinks) {
+      store_root_.addSink(*sink);
+    }
+
+    // Default flush interval is 5s.
```

Comment:
Does this comment help, or is it likely to get out of sync with the real default value (if that ever gets changed) and confuse the reader?

Reply:
Yes, it could be out of sync in the future and the information about the default value is not very useful here, so removed this comment.